package.json

```diff
@@ -3,25 +3,24 @@
   "version": "0.0.5",
   "description": "Hapi.js plugin for authentication scheme of accepting connections only from certain IPs",
   "main": "lib/index.js",
-  "typings": "lib/index.d.ts",
   "repository": "https://github.com/chamini2/hapi-auth-ip-whitelist",
   "author": "Matteo Ferrando <matteo.ferrando2@gmail.com>",
   "license": "MIT",
-  "scripts": {
-    "build": "tsc",
-    "prepublish": "npm run build"
-  },
   "devDependencies": {
-    "@types/boom": "^4.3.7",
-    "@types/hapi": "^16.1.12",
-    "@types/ip": "^0.0.30",
-    "typescript": "^2.6.2"
+    "hapi": "^17.2.0",
+	"therealyou": "^2.0.0"
   },
   "dependencies": {
-    "boom": "^5.2.0",
+    "boom": "^7.1.1",
     "ip": "^1.1.5"
   },
   "peerDependencies": {
-	"hapi": "< 17"
+    "hapi": ">= 17"
+  },
+  "optionalDependencies": {
+	"therealyou": "^2.0.0"
```

Comment:
I would remove this.

Reply:
Maybe you are right, it's not really a dependency.